utils/build_swift/postbuild_steps.py

```diff
@@ -0,0 +1,115 @@
+#===--- postbuild.py -----------------------------------------------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https:#swift.org/LICENSE.txt for license information
+# See https:#swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+
+import os
+import platform
+import shutil
+
+import swift_build_support.swift_build_support.products as products
```

Comment:
Same here, we should stick with the same `from package import module` syntax rather than qualified imports like this.